src/test/suite/connectionController.test.ts

```diff
@@ -889,14 +900,20 @@ suite('Connection Controller Test Suite', function () {
       const currentConnectingVersion = testConnectionController.getConnectingVersion();
 
       assert(currentConnectingVersion !== null);
-      assert(currentConnectingVersion === testConnectionController._connections[
-        Object.keys(testConnectionController._connections)[0]
-      ].id);
+      assert(
```

Comment:
`assert.equal` or `assert.strictEqual` will give you a better error message and output a diff